extmod/modure.c

```diff
@@ -405,7 +405,7 @@ STATIC MP_DEFINE_CONST_DICT(re_locals_dict, re_locals_dict_table);
 
 STATIC const mp_obj_type_t re_type = {
     { &mp_type_type },
-    .name = MP_QSTR_ure,
+    .name = MP_QSTR_re,
```

Comment:
#if CIRCUITPY this too if you are doing it elsewhere.

Reply:
Oops, yes.